### ISSUE TITLE
Unable to resolve @import "./base.css"

### ISSUE THREAD

**Issue by chansee97** [sxzz/ast-explorer#69]
### Describe the bug

After start, `composables\parser\template\css.css` There is an error in the reference to this file.
```css
// Unable to resolve @import "./base.css"
@import './base.css';
```

### Reproduction

none

### System Info

```Shell
none
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/sxzz/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/sxzz/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sxzz** [comment]
I believe this is an issue caused upstream, and I have already submitted it to nuxt
https://github.com/nuxt/nuxt/issues/27753

**Comment by sxzz** [comment]
Temporarily fixed via aa62dcb56364d35dcfef2e3d7fcfd9d678ad7d0f
